docs/CommonMethods/spin_testing.md

```diff
@@ -3,3 +3,37 @@ layout: default
 title: Spin Tests
 parent: Common Methods
 ---
+
+# Spin tests
+
+If there is one statistical method that the lab pretty much always uses at one point or another, it’s `spin tests`!! 
+
+What’s a spin test, you might ask? Briefly, it’s a test that allows to determine whether say two brain maps are significantly associated. The trick here is that this type of test accounts for the fact that brain regions that are close together tend to share similar brain features (i.e., there is spatial autocorrelation in brain data). The spin test assesses significance by transforming the brain into a little sphere and rotating one of the two original brain maps a bunch (let’s say, 10,000) times. You then recompute your test statistic (e.g., a correlation between the two maps), except that this time it’s a correlation between a spun map and the second (unchanged) brain map. This generates a “null distribution” of your test statistic, which represents results that you could have obtained just due to simple autocorrelation in the brain data. 
```

Comment:
Can you split sentences into separate lines? Modifying and diffing long lines can be annoying.